chapters/crud/index.md

```diff
@@ -1,24 +1,40 @@
 # CRUD Operations
 
-CRUD stands for Create, Read, Update, Delete. Together, they form the basis of any database engine. While Jina itself is *not* a database engine, it does provide these methods of interacting with the data stored in the indexers.
+##### In this section, you will get an overview of how Jina implements CRUD: Create, Read, Update, Delete  
 
-Until now, Jina has only supported indexing (creating) and querying (reading) documents. To update or delete a document, you'd have to edit your dataset, and then re-build the Flow and indexers. Needless to say, this could create problems if you had large datasets.
+### Feature description
 
-With the release of version `1.0` we're introducing **update** and **delete** operations. These are implemented across our executors and drivers and will allow you to update and delete documents by their ids. 
+CRUD stands for Create, Read, Update, Delete. Together, they form the basis of any database engine. While Jina itself is *not* a database engine, it does provide these methods of interacting with the data stored in its indexes.
+
+Before `1.0`, Jina only supported indexing (creating) and querying (reading) Documents. To update or delete a Document, you had to edit your dataset, and then rebuild the Flow and Indexers. Needless to say, this could create problems if you had large datasets.
+
+With the release of version `1.0` we are introducing **update** and **delete** operations. These are implemented across our Executors and Drivers, and allow you to update and delete Documents by their ids. 
+
+## Before you start 
+
+You will need to know the basics of Jina. See [here](https://docs.jina.ai/chapters/core/introduction/index.html)
+
+### Implementation
 
 A basic example of this can be found in the `test_crud.py` file under `tests/integration/crud/simple`:
 
-The Flow object now supports a `delete` and `update` method, with a signature similar to index:
+The Flow class now supports a `delete` and `update` methods, with a signature similar to index:
 
 ```python
+    docs = random_docs(10)
```

Comment:
can you import `random_docs` and `flow` in the code samples? to allow user able to copy-paste this piece of code block and run it

Reply:
I think the philosophy is to keep the code in docs to be minimal and illustrative, not fully-fledged runnable code.

Reply:
- supports ~a~ `delete` and `update` methods (remove the `a`)